package/src/components/MessageInput/MessageInput.tsx

```diff
@@ -418,10 +439,13 @@ const MessageInputWithContext = <
       }
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [imageUploadsLength, hasImagePicker]);
+  }, [imageUploadsLength]);
```

Comment:
same here, remove and lets fix them

Reply:
As we decided this would be a big change on our side, so we will look into this later. Thanks 🙂